README.md

```diff
@@ -30,6 +30,11 @@ To start writing
 smart contracts, the best way to go is to use the [Atom editor](https://atom.io) and install
 `language-pact` using the [package manager](http://flight-manual.atom.io/using-atom/sections/atom-packages/).
 
+Quick Start: Vim
+---
+
+The [vim-pact](https://github.com/wsdjeg/vim-pact) provides pact supporting for [vim editor](https://vim.org), 
```

Comment:
Typo comma at end. Also on the vim-pact project itself, an installation guide in the readme might be a good idea.

Reply:
Seconded on the install how-to. Otherwise, this looks great!